src/handbook/development/markdown-how-to.md

```diff
@@ -100,3 +109,15 @@ this will write the content as if it is code
 this will write the content as if it is code
 ```
 ````
+
+Add syntax highlighting by defining the language, for example:
+
+```js
+console.log("Hello, world!")
+```
+
+```md
+```js
+console.log("Hello, world!")
+```
+```
```

Comment:
`````suggestion
````md
```js
console.log("Hello, world!")
```
````
`````

To have backticks inside backticks you need more backticks... it's like boring Inception